learning_resources/etl/utils.py

```diff
@@ -395,14 +332,50 @@ def documents_from_olx(
                 yield (
                     filebytes,
                     {
-                        "key": f"document_{next(counter)}_{filename}",
+                        "key": f"{path}/{filename}",
                         "content_type": CONTENT_TYPE_FILE,
                         "mime_type": mimetype,
                         "checksum": md5(filebytes).hexdigest(),  # noqa: S324
+                        "file_extension": extension_lower,
+                        "source_path": path,
                     },
                 )
 
 
+def clean_srt_content(content: str):
+    """
+    Remove timestamps and other extraneous data from SRT content
+
+    Args:
+        content (str): The SRT content to clean
+
+    Returns:
+        str: The content as a string without timestamps
+    """
+    # Remove timestamps
+    content = re.sub(
+        r"\d{2}:\d{2}:\d{2},\d{3} --> \d{2}:\d{2}:\d{2},\d{3}\n", "", content
+    )
+    # Remove sequence numbers
+    content = re.sub(r"\d+\n", "", content)
+    # Remove empty lines
+    return re.sub(r"\n\s*\n", " ", content)
+
+
+def clean_sjson_content(content: str):
+    """
+    Remove timestamps  from sjson content
```

Comment:
may want to clarify the docstring since it seems like this method returns the text content rather than removing anything. (minor nitpick - seems like some extra spaces in the docstring)